unpackaged/config/trial_translations/objectTranslations/Opportunity-de.objectTranslation

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObjectTranslation xmlns="http://soap.sforce.com/2006/04/metadata">
+  <recordTypes>
+    <description>Spende eingegangen</description>
+    <label>Spende</label>
+    <name>Donation</name>
+  </recordTypes>
+  <recordTypes>
+    <description>Fördergelder eingegangen</description>
+    <label>Fördergelder</label>
+    <name>Grant</name>
+  </recordTypes>
+  <recordTypes>
+    <description>Als Sachspende eingegangene Waren oder Dienstleistungen</description>
+    <label>Sachgeschenk</label>
+    <name>InKindGift</name>
+  </recordTypes>
+  <recordTypes>
+    <description>Große Spende von einem Großspender</description>
+    <label>Großschenkung</label>
+    <name>MajorGift</name>
+  </recordTypes>
+  <recordTypes>
+    <description>Zugewiesene Spende vom Arbeitgeber eines Spenders oder einer anderen Organisation</description>
+    <label>Zugewiesene Schenkung</label>
+    <name>MatchingGift</name>
+  </recordTypes>
+  <recordTypes>
+    <description>Ein Mitgliedschaft-Datensatztyp</description>
+    <label>Mitgliedschaft</label>
+    <name>Membership</name>
+  </recordTypes>
+  <layouts>
+    <layout>___NAMESPACE___Major Gift Layout</layout>
```

Comment:
The token `___NAMESPACE___` is replaced only in filenames. In the file body, it needs to be `%%%NAMESPACE%%%`.

Reply:
Thanks. Changes made.